_search-plugins/cross-cluster-search.md

```diff
@@ -167,6 +164,9 @@ curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-passwo
   }
 }'
 ```
+All of the curl requests can also be sent using OpenSearch Dashboards Dev Tools.
+{: .tip }
+![OpenSearch Dashboards UI for configuring remote cluster for Cross-cluster search]({{site.url}}{{site.baseurl}}/images/ccs-devtools.png)
```

Comment:
Please reference in the preceding text.

Reply:
I've now added a line before the image, If thats what you meant @natebower

Reply:
@AntonEliatra Yes, correct. See my edits.